src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,15 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import classnames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={classnames('TodoInfo',
+    { 'TodoInfo--completed': todo.completed })}
+  >
+    <h2 className="TodoInfo__title">
+      {todo.title}
```

Comment:
After when first comment will implemented
```suggestion
      {title}
```